### ISSUE TITLE
`format` in config files is ignored

### ISSUE THREAD

**Issue by bendrucker** [terraform-linters/tflint#2305]
When `format` is set in a config file, it doesn't do anything. Only the `--format` flag works.

```hcl
config {
  format = "compact"
}
```

```sh
tflint --config config.hcl
```

```
1 issue(s) found:

Warning: terraform "required_version" attribute is required (terraform_required_version)

  on main.tf line 1:

Reference: https://github.com/terraform-linters/tflint-ruleset-terraform/blob/v0.10.0/docs/rules/terraform_required_version.md
```

```sh
tflint --format compact
```

```
1 issue(s) found:

main.tf:1:1: Warning - terraform "required_version" attribute is required (terraform_required_version)
```
